1-click.go

```diff
@@ -33,6 +34,17 @@ type OneClicksRoot struct {
 	List []*OneClick `json:"1_clicks"`
 }
 
+// InstallKubernetesApps represents a request required to install 1-click kubernetes apps
+type InstallKubernetesApps struct {
```

Comment:
`InstallKubernetesAppsRequest` might be a bit more consistent with other services.

Reply:
Made the change.